legend-sdlc-server/src/test/java/org/finos/legend/sdlc/server/project/TestProjectStructure.java

```diff
@@ -296,13 +311,13 @@ public void testUpdateProjectDependencies_Prototype()
     protected void testUpdateProjectDependencies(ProjectType projectType)
```

Comment:
Could you make sure that some test (either this one or a new one) demonstrates upgrading old-style dependencies to new-style?